demo/admin/webpack.config.ts

```diff
@@ -133,6 +137,10 @@ const config = ({ production }: IEnvironment): webpack.Configuration => {
             alias: {
                 "@src": path.resolve(__dirname, "src/"),
             },
+            fallback: {
+                "react/jsx-runtime": "react/jsx-runtime.js",
+                "react/jsx-dev-runtime": "react/jsx-dev-runtime.js",
+            },
```

Comment:
is needed because react-dnd >=16 needs react 18, this is the correct fallback

https://github.com/vivid-planet/comet/pull/705